en/02-preparing-your-project-being-open-sourced.md

```diff
@@ -1,70 +1,67 @@
+Si votre projet est sur GitHub, ce fichier sera [lié automatiquement](https://help.github.com/en/articles/setting-guidelines-for-repository-contributors) lorsqu'un contributeur ouvre un problème ou crée une demande de tirage.
 
 ## Code of conduct
 
-Having a CODE_OF_CONDUCT.md in your public repository lets potential _contributors_ know in advance how they can expect to be treated by the community and _maintainers_. -- [Michael Jolley](https://dev.to/michaeljolley/using-a-contributing-codeofconduct-to-assist-others-in-contributing-to-public-repositories-1l90)
-
-The code of conduct of your project is a document that protects every participant. It helps creating a welcoming community.
+Le fait d'avoir un CODE_OF_CONDUCT.md dans votre référentiel public permet aux potentiels contributeurs de savoir à l'avance comment ils peuvent s'attendre à être traités par la communauté et les mainteneurs. - [Michael Jolley](https://dev.to/michaeljolley/using-a-contributing-codeofconduct-to-assist-others-in-contributing-to-public-repositories-1l90)
 
-A good addition to your code of conduct is an explanation about how you plan to enforce it. It's important to show that you take it seriously so everybody knows what action will be taken in case of a code of conduct violation.
+Le code de conduite de votre projet est un document qui protège chaque participant. Cela aide à créer une communauté accueillante.
 
-You should also clarify the way to report a violation, such as through an email.
+Un bon ajout à votre code de conduite est une explication de la manière dont vous envisagez de le faire respecter. Il est important de montrer que vous prenez cela au sérieux afin que tout le monde sache quelles mesures seront prises en cas de violation du code de conduite.
 
-The most famous code of conduct is [Contributor Covenant](https://www.contributor-covenant.org/). It is used by thousands of open source projects. If your project is on GitHub you can directly add Contributor Covenant or [Citizen Code of Conduct](http://citizencodeofconduct.org/) through your repository interface. You need to go to `Insights > Community > Code of conduct`, then by clicking on the Add button and choosing a code of conduct, it will create a commit for you.
+Vous devez également préciser la manière de signaler une violation, par exemple par courrier électronique.
+Le code de conduite le plus connu est [contributeur](https://www.contributor-covenant.org/). Il est utilisé par des milliers de projets open source. Si votre projet est sur GitHub, vous pouvez directement ajouter le covenant du contributeur ou le [code de conduite du citoyen].(http://citizencodeofconduct.org/) via votre interface de référentiel. Vous devez aller dans «Insights> Communauté> Code de conduite», puis en cliquant sur le bouton Ajouter et en choisissant un code de conduite, un commit sera créé pour vous.
```

Comment:
Contributor Covenant et Citizen Code of Conduct being organizations name, keep them in english.